x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/layer_panel.tsx

```diff
@@ -193,112 +195,153 @@ export function LayerPanel(
                             columnId: accessor,
                             filterOperations: group.filterOperations,
-                            suggestedPriority: group.suggestedPriority,
-                            onClick: () => {
-                              if (activeId) {
-                                setActiveDimension(initialActiveDimensionState);
-                              } else {
-                                setActiveDimension({
-                                  isNew: false,
-                                  activeGroup: group,
-                                  activeId: accessor,
-                                });
-                              }
-                            },
-                          }}
-                        />
-                        <EuiButtonIcon
-                          data-test-subj="indexPattern-dimension-remove"
-                          iconType="cross"
-                          iconSize="s"
-                          size="s"
-                          color="danger"
-                          aria-label={i18n.translate('xpack.lens.indexPattern.removeColumnLabel', {
-                            defaultMessage: 'Remove configuration',
-                          })}
-                          title={i18n.translate('xpack.lens.indexPattern.removeColumnLabel', {
-                            defaultMessage: 'Remove configuration',
-                          })}
-                          onClick={() => {
-                            trackUiEvent('indexpattern_dimension_removed');
-                            props.updateAll(
-                              datasourceId,
-                              layerDatasource.removeColumn({
-                                layerId,
-                                columnId: accessor,
-                                prevState: layerDatasourceState,
-                              }),
+                          });
+                          if (typeof dropResult === 'object') {
+                            // When a column is moved, we delete the reference to the old
+                            props.updateVisualization(
                               activeVisualization.removeDimension({
                                 layerId,
-                                columnId: accessor,
+                                columnId: dropResult.deleted,
                                 prevState: props.visualizationState,
                               })
                             );
-                          }}
-                        />
-                      </div>
-                    </DragDrop>
-                  );
-                })}
+                          }
+                        }}
+                      >
+                        <div className="lnsLayerPanel__dimension">
+                          <NativeRenderer
+                            render={props.datasourceMap[datasourceId].renderDimensionTrigger}
```

Comment:
There's an issue with the styling of this element in Firefox: the non-link properties aren't draggable:

![drag to reorder issues in firefox](https://user-images.githubusercontent.com/666475/96926702-ea5a7480-1483-11eb-9c56-af6a3edc5ff6.gif)

Reply:
Thanks, we have this bug since forever here, nice testing! 
This is caused by the bug that exists in Firefox for 11 years 😅  https://bugzilla.mozilla.org/show_bug.cgi?id=568313
It was easy to fix by surrounding the button content with <EuiFlexItem> </EuiFlexItem> tags.

Reply:
Thanks!